tests/testJointcal.py

```diff
@@ -10,7 +10,7 @@
 import eups
 
 from lsst.afw import geom, coord, table
-from lsst.meas.astrom import astrometry
+from lsst.meas import astrom
```

Comment:
Done. (though "from lsst.meas import astrom" shouldn't pollute the namespace at all.)